packages/perseus-core/src/utils/get-divide-symbol.ts

```diff
@@ -0,0 +1,44 @@
+const localesThatUseColon = [
+    "az",
+    "bg",
+    "cs",
+    "da",
+    "de",
+    "hu",
+    "hy",
+    "id",
+    "it",
+    "ky",
+    "lt",
+    "lv",
+    "nb",
+    "nl",
+    "pl",
+    "pt-pt",
+    "ro",
+    "sv",
+    "uk",
+    "vi",
+    "ro",
+    "ru",
+];
+
+/**
+ *  Get the character used for dividing numbers.
+ */
+export const getDivideSymbol = (locale: string): string => {
+    // If the locale uses a colon for division, return a colon
+    if (localesThatUseColon.includes(locale)) {
+        return ":";
+    }
+
+    // Default to using a forward slash
+    return "/";
+};
+
+export const getDivideSymbolForTex = (locale: string): string => {
```

Comment:
We have tests that required the keypad to input _tex_ commands, but we need to use the default slash for scoring logic. This is one idiosyncrasy that the division symbol has when compared to the decimalSeparator, which doesn't require tex. 

I opted to make a separate function for supporting the keypad, that uses the same logic as getDivideSymbol. I'm happy to take any alternative suggestions on this, however!

Reply:
Makes sense to me. It could be helpful to put this explanation in a doc comment for `getDivideSymbolForTex`